src/views/SubmissionView/hooks/useSubmissionValidationStatus.test.js

```diff
@@ -13,26 +12,31 @@ jest.mock('hooks/app', () => ({
   useTextResponses: jest.fn(),
 }));
 
-const state = mockUseKeyedState(stateKeys);
-
 describe('useSubmissionValidationStatus', () => {
   useSubmissionConfig.mockReturnValue({
     textResponseConfig: { required: false },
     fileResponseConfig: { required: false },
   });
 
+  let setStateSpy;
+  const setValue = jest.fn();
+
   beforeEach(() => {
+    setStateSpy = jest.spyOn(React, 'useState').mockImplementation((value) => [value, setValue]);
+  });
+
+  afterEach(() => {
+    setStateSpy.mockRestore();
     jest.clearAllMocks();
-    state.mock();
   });
-  afterEach(() => { state.resetVals(); });
 
   it('initializes state for useSubmissionValidationStatus', () => {
     useSubmissionConfig.mockReturnValue();
     useSubmissionValidationStatus([], []);
-    state.expectInitializedWith(stateKeys.submissionTriggered, false);
-    state.expectInitializedWith(stateKeys.promptStatuses, {});
-    state.expectInitializedWith(stateKeys.fileUploadIsRequired, false);
+    expect(setStateSpy).toHaveBeenCalledTimes(3);
+    expect(setStateSpy).toHaveBeenCalledWith({});
+    expect(setStateSpy).toHaveBeenNthCalledWith(2, false);
+    expect(setStateSpy).toHaveBeenNthCalledWith(3, false);
```

Comment:
It'd be great to add comments here similar to the ones added in `FileRenderer/hooks.test.js`.